### PR TITLE
Poloniex Patch Part 1

### PR DESCRIPTION
**Pull request by TheRoboKitten**
Poloniex fixes and python scripts and index pages for apache2

### CODE REVIEW

**Comment by crubb** [comment]
These seem to be different things you're trying to merge. Auto Backtester stuff as well as polo API stuff. You'll need to separate them before submitting to carlos.

File: package.json [hunk 1]
```diff
@@ -27,7 +27,7 @@
     "node.bittrex.api": "^0.2.5",
     "number-abbreviate": "^2.0.0",
     "numbro": "git+https://github.com/carlos8f/numbro.git",
-    "poloniex.js": "0.0.7",
+    "poloniex.js": "git+https://github.com/premasagar/poloniex.js.git",
```
Comment (crubb) [inline package.json]: A git Repo is a running target. Can you fix this to a version/a commit?

File: lib/engine.js [hunk 2]
```diff
@@ -457,7 +457,7 @@ module.exports = function container (get, set, clear) {
                     return cb(err)
                   }
                 }
-                if (n(s.balance.asset).subtract(s.balance.asset_hold || 0).value() < n(size).value()) {
+                if (n(s.balance.asset || 0).value() < n(s.balance.asset_hold).value()) {
```
Comment (crubb) [inline lib/engine.js]: I had problems like this with kraken. Look at the kraken exchange.js. I don't think it's a good idea to change the logic of zenbot here but rather do the proper things/returns in the corresponding exchange.js.

Comment (TheRoboKitten) [inline lib/engine.js]: I understand what you mean.

**Pull request closed without merging**